examples/python/route_guide/run_codegen.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
Optional: we could use "-x" on this script, so users can find out what it is doing without actually open it.

Reply:
Probably a good idea in general, but the Python protoc plugin actually results in a 0 exit code in case of failure by design. (not sure why that decision was made)

Reply:
LOL. I tried to change it to non-zero at one point, and our CI failed. Our test protos compilation triggers protoc error, but I failed to trace the root cause. So, I guess it's a tech debt never got paid.

I meant to turn on "xtrace" mode, not error on non-zero exit code.